packages/app-cli/app/command-mkbook.test.ts

```diff
@@ -0,0 +1,56 @@
+import { setupDatabaseAndSynchronizer, switchClient } from '@joplin/lib/testing/test-utils';
+import { setupCommandForTesting, setupApplication } from './utils/testUtils';
+import Folder from '@joplin/lib/models/Folder';
+const Command = require('./command-mkbook');
+
+
+describe('command-mkbook', function() {
+
+	beforeEach(async () => {
+		await setupDatabaseAndSynchronizer(1);
+		await switchClient(1);
+		await setupApplication();
+	});
+
+
+	it('should create a subfolder in first folder', async () => {
+		const command = setupCommandForTesting(Command);
+		await expect(command.action({ 'new-notebook': 'folder1' })).resolves.not.toThrowError();
+		await expect(command.action({ 'new-notebook': 'folder1_1' , options: { s: true, sub: true } })).resolves.not.toThrowError();
```

Comment:
I don't think you need to specify both -s and --sub. Please only use the long form --sub